marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-view.test.tsx

```diff
@@ -150,4 +152,58 @@ describe("Graph View Component", () => {
 
     expect(getByLabelText("publish-to-database")).toBeDisabled();
   });
-});
\ No newline at end of file
+
+  test("Validates modeling Graph UI text ", async () => {
+
+    const mockDeleteEntity = jest.fn();
+
+    const {getByText, getByLabelText, queryByLabelText, rerender, getAllByLabelText} =  render(
+      <ModelingContext.Provider value={isModified}>
+        <GraphView
+          entityTypes={getEntityTypes}
+          canReadEntityModel={true}
+          canWriteEntityModel={true}
+          deleteEntityType={mockDeleteEntity}
+          relationshipModalVisible={false}
+          toggleRelationshipModal={jest.fn()}
+          updateSavedEntity={jest.fn()}
+          setEntityTypesFromServer={jest.fn()}
+          hubCentralConfig={hubCentralConfig}
+          updateHubCentralConfig={jest.fn()}
+        />
+      </ModelingContext.Provider>
+    );
+
+    expect(queryByLabelText("Product-selectedEntity")).not.toBeInTheDocument();
+
+    // Opens side panel
+    rerender(withEntityAs("Product"));
+    await wait(() => expect(getByLabelText("Product-selectedEntity")).toBeInTheDocument());
+
+    // Verifies side panel content
+    const entityTypeTab = getByText("Entity Type");
+    expect(entityTypeTab).toBeInTheDocument();
+    act(() => {
+      userEvent.click(entityTypeTab);
+    });
+
+    // Hovers over color question icon to see tooltip
+    const tooltips = getAllByLabelText("icon: question-circle");
+    act(() => {
+      // color tooltip has index 1 in the array
+      userEvent.hover(tooltips[1]);
+    });
+    expect(document.querySelector("[class='tooltip-inner']")?.firstChild?.textContent).toEqual("Select a color to associate it with the Product entity throughout your project.");
+
+    // Clicks on Add dropdown
+    const addDropdown = getByText("Add");//getByLabelText("add-entity-type-relationship");
```

Comment:
@littlecherro  This test looks excellent! Let's remove this duplicate assertion that's commented out if it is unneeded.